lib/index.js

```diff
@@ -342,9 +342,22 @@ internals.addMethod(['instanceof', 'instanceOf'], internals.instanceof);
 
 internals.match = function (regex) {
 
+    if (this._ref instanceof Array) {
+
+        return this.assert(internals.matchArray(this._ref, regex), 'match ' + regex);
+    }
     return this.assert(regex.exec(this._ref), 'match ' + regex);
 };
 
+internals.matchArray = function (items, regex) {
```

Comment:
I don't think this needs to be a separate function. You can just do the `some()` in `match()`